src/main/java/org/jenkinsci/plugins/GithubRequireOrganizationMembershipACL.java

```diff
@@ -343,6 +366,7 @@ public GithubRequireOrganizationMembershipACL(List<String> adminUserNameList,
         super();
 
         this.adminUserNameList                    = adminUserNameList;
+        this.agentUserName                   = agentUserName;
```

Comment:
To align the `=` for style consistency:

```suggestion
        this.agentUserName                        = agentUserName;
```